src/main/java/core/lib/Injector.java

```diff
@@ -0,0 +1,45 @@
+package core.lib;
+
+import core.dao.BetDao;
+import core.dao.BetDaoImp;
+import core.dao.GainDao;
+import core.dao.GainDaoImpl;
+import core.exception.NotExistAnnotationException;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class Injector {
+    public static Object getInstance(Class clazz) {
+        Constructor constructor;
+        Object instance;
+        Field[] declaredFields = clazz.getDeclaredFields();
+        try {
+            constructor = clazz.getConstructor();
+            instance = constructor.newInstance();
+            for (Field field : declaredFields) {
+                if (field.getAnnotation(Inject.class) != null) {
+                    field.setAccessible(true);
+                    if (field.getType().equals(BetDao.class)
+                            && BetDaoImp.class.getAnnotation(Dao.class) != null) {
+                        field.set(instance, new BetDaoImp());
+                    } else if (BetDaoImp.class.getAnnotation(Dao.class) == null) {
+                        throw new NotExistAnnotationException("Not exist annotation "
```

Comment:
you can put throwing an exception into else clause so the code won't repeat itself